projects/mcp-server.md

```diff
@@ -0,0 +1,136 @@
+# OpenTelemetry Collector Model Context Protocol (MCP) Server
+
+## Background and description
+
+The OpenTelemetry project consists of a large number of components, including collector, SDKs, and instrumentation libraries, which are often configured and managed separately. This distribution of components poses a major operational challenge which is universally recognized by the community [1], [2].
+
+Large language models (LLMs), present a significant opportunity to simplify the adoption, implementation, and management of the OpenTelemetry stack. An AI agent could, for example, analyze telemetry data, facilitate configuration changes, or assist and simplify the instrumentation process.
+
+However, to support this process, a standardized interface is required for LLMs to interact with the OpenTelemetry ecosystem. The Model Context Protocol (MCP) provides an idiomatic approach for this interaction.
+
+At the moment, the OpenTelemetry project does not have an official, standard MCP server. This has led to the creation of several independent, open-source projects to fill the gap.
+
+### Existing OpenTelemetry MCP Servers
+
+The proliferation of these projects demonstrates strong community interest and the clear potential of this technology:
+
+* [pavolloffay/opentelemetry-mcp-server](https://github.com/pavolloffay/opentelemetry-mcp-server): Focuses on collector configuration.
+* [austinlparker/otel-mcp](https://github.com/austinlparker/otel-mcp): Handles collector configuration and data profiling.
+* [mottibec/otelcol-mcp](https://github.com/mottibec/otelcol-mcp): Focuses on collector configuration.
+* [shiftyp/otel-mcp-server](https://github.com/shiftyp/otel-mcp-server): Provides data profiling, but requires OpenSearch.
+* [liatrio-labs/otel-instrumentation-mcp](https://github.com/liatrio-labs/otel-instrumentation-mcp): Manages instrumentation.
+* [traceloop/opentelemetry-mcp-server](https://github.com/traceloop/opentelemetry-mcp-server): Provides data profiling by connecting to Jaeger, Tempo and Traceloop.
+
+Each of these servers uses a different approach, particularly for collector configuration and data profiling.
+This fragmentation creates confusion for users in terms of installation and configuration. It is less effective
+as multiple competing tools fill up context window and provide overlapping functionality.
+
+### Project Scope and Architecture
```

Comment:
Add 2 things that are out of scope:
* Any MCP server that works on top of opentelemetry data, as this is a "backend concern"
* MCP servers should not have a shadow knowledge base or documentation, they will pull these informations from docs, repos, and [ecosystem explorer](https://github.com/open-telemetry/community/pull/3000)

Reply:
I have documented this in the `### Non Goals` section